### DISCUSSION TITLE
outdated info?

### DISCUSSION THREAD

**Discussion by kwit98** [espressif/arduino-esp32#8947] [General]
In one place u wrote that ESP32-S series is being categorized by not having BLE, but in data sheet on S3 series all (most?) have them:

https://github.com/espressif/arduino-esp32/blob/master/docs/source/boards/boards.rst

"To help you on this selection, we point out some facts about choosing the proper boards to help you to save money and time  [...]

The ESP32 is divided by family:

ESP32
Wi-Fi and BLE

ESP32-S
Wi-Fi only

ESP32-C
Wi-Fi and BLE 5

ESP32-H
BLE and IEEE 802.15.4

For each family, we have SoC variants with some differentiation. The differences are more about the embedded flash and its size and the number of the cores (dual or single). "

---------------------------------------------

https://www.espressif.com/sites/default/files/documentation/esp32-s3_datasheet_en.pdf

In multiple places, but for example on page 3

**Comment by P-R-O-C-H-Y** [comment]
Hi @kwit98, you are right. As more and more of new chips comes, these statements or divide is no more accurate.
As you wrote the ESP32-S3 has the BLE 5 and also ESP32-C6 has the IEEE 802.15.4.
We will probably need to take a look and rework this. Thank you for pointing that out :)
